docs/rules/no-bare-urls.md

```diff
@@ -0,0 +1,53 @@
+# no-bare-urls
+
+Disallow bare URLs.
+
+## Background
+
+In Markdown, URLs are typically formatted in two ways:
+
+1. Autolinks: URLs wrapped in angle brackets, such as `<https://example.com/>`
+2. Links: URLs with descriptive text, such as `[Visit our website](https://example.com/)`
+
+A bare URL appears without angle brackets (`<>`), such as `https://example.com/`. While GitHub Flavored Markdown (GFM) allows bare URLs to be recognized as clickable links, this feature is not supported in all Markdown processors. To ensure compatibility across different processors, use autolinks or links instead.
```

Comment:
Can you link "autolinks" and "links" in the last sentence to somewhere? I'm not sure people will understand what these mean without additional context.